src/loop_libraries/library_collier.cpp

```diff
@@ -0,0 +1,165 @@
+   const std::complex<double> m22 = m22_in;\
+   const std::complex<double> m32 = m32_in;\
+\
+   set_mu2_uv(scl2_in);\
+   return BOOST_PP_CAT(NAME,_impl)(&p10, &p21, &p32, &p30, &p20, &p31, &m02, &m12, &m22, &m32);\
+}
+
+// Fortran wrapper routines
+extern "C" {
+   void initialize_collier_impl();
+   void set_mu2_uv_impl(double*);
+
+   BOOST_PP_SEQ_FOR_EACH(IMPL,0,A_SEQ)
+   BOOST_PP_SEQ_FOR_EACH(IMPL,2,B_SEQ)
+   BOOST_PP_SEQ_FOR_EACH(IMPL,5,C_SEQ)
+   BOOST_PP_SEQ_FOR_EACH(IMPL,9,D_SEQ)
+
+   void get_A_impl(const std::complex<double> [1], COLLIER_ARGS(0));
+   void get_B_impl(const std::complex<double> [2], COLLIER_ARGS(2));
+   void get_C_impl(const std::complex<double> [7], COLLIER_ARGS(5));
+   void get_D_impl(const std::complex<double> [11], COLLIER_ARGS(9));
+}
+
+namespace looplibrary
+{
+
+void Collier::initialize() noexcept
+{
+   initialize_collier_impl();
+}
+
+void Collier::set_mu2_uv(double scl2_in) noexcept
+{
+   double scl2 = scl2_in;
+   if( std::abs(scl2 - this->current_mu2_uv) > std::numeric_limits<double>::epsilon() )
+   {
+      set_mu2_uv_impl(&scl2);
+      this->current_mu2_uv = scl2;
+   }
+}
+
+std::complex<double> Collier::A0(A_ARGS) noexcept
+{
+   const std::complex<double> m02 = m02_in;
+
+   set_mu2_uv(scl2_in);
+   return A0_impl(&m02);
+}
+
+BOOST_PP_SEQ_FOR_EACH(COLLIER_B,(B_ARGS),B_SEQ)
+BOOST_PP_SEQ_FOR_EACH(COLLIER_C,(C_ARGS),C_SEQ)
+BOOST_PP_SEQ_FOR_EACH(COLLIER_D,(D_ARGS),D_SEQ)
+
+void Collier::A(std::complex<double> (&a)[1], A_ARGS) noexcept
+{
+   const std::complex<double> m02 = m02_in;
+
+   set_mu2_uv(scl2_in);
+   get_A_impl(a, &m02);
+}
+
+void Collier::B(std::complex<double> (&b)[2], B_ARGS) noexcept
+{
+   const std::complex<double> p10 (p10_in.real(), 0.);
+   const std::complex<double> m02 = m02_in;
+   const std::complex<double> m12 = m12_in;
+
+   set_mu2_uv(scl2_in);
+   get_B_impl(b, &p10, &m02, &m12);
+}
+
+void Collier::C(
+   std::complex<double> (&c)[7], C_ARGS) noexcept
+{
+   const std::complex<double> p10 (p10_in.real(), 0.);
+   const std::complex<double> p21 (p21_in.real(), 0.);
+   const std::complex<double> p20 (p20_in.real(), 0.);
+   const std::complex<double> m02 = m02_in;
+   const std::complex<double> m12 = m12_in;
+   const std::complex<double> m22 = m22_in;
+
+   set_mu2_uv(scl2_in);
+   get_C_impl(c, &p10, &p21, &p20, &m02, &m12, &m22);
+}
+
+void Collier::D(
+   std::complex<double> (&d)[11], D_ARGS) noexcept
+{
+   const std::complex<double> p10 (p10_in.real(), 0.);
+   const std::complex<double> p21 (p21_in.real(), 0.);
+   const std::complex<double> p32 (p32_in.real(), 0.);
+   const std::complex<double> p30 (p30_in.real(), 0.);
+   const std::complex<double> p20 (p20_in.real(), 0.);
+   const std::complex<double> p31 (p31_in.real(), 0.);
+   const std::complex<double> m02 = m02_in;
+   const std::complex<double> m12 = m12_in;
+   const std::complex<double> m22 = m22_in;
+   const std::complex<double> m32 = m32_in;
+
+   set_mu2_uv(scl2_in);\
```

Comment:
please remove the `\` at the end of the line

Reply:
Done in 740d0f5.